face_qa/face_qa.py

```
@@ -113,101 +102,138 @@ def _face_detection_v2(self) -> bool:
+
+            mouth_width = np.linalg.norm(right_pt - left_pt)
+            mouth_height = np.linalg.norm(top_pt - bottom_pt)
+            smile_ratio = mouth_width / (mouth_height + 1e-6)
+
+
+
+            # Validações extras
+            if mouth_width < min_mouth_width:
+                smiling = False
+            elif smile_ratio < 1.1:
+                smiling = False
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)